addons/l10n_hk/__manifest__.py

```diff
@@ -7,12 +6,13 @@
     'category': 'Accounting/Localizations/Account Charts',
     'description': """ This is the base module to manage chart of accounting and localization for Hong Kong """,
     'author': 'Odoo SA',
-    'depends': ['account'],
+    'depends': ['account_qr_code_emv'],
```

Comment:
You cannot modify the dependencies of a module in stable :/

Reply:
:cry: May I ask what should I do in this case? Make the module account_qr_code_emv auto install? Or I can only target master?

Reply:
`account_qr_code_emv` could be auto installed if `account` and `l10n_hk` are installed for instance yes

Here's an example: https://github.com/odoo/odoo/blob/16.0/addons/l10n_it_edi/__manifest__.py#L15

Reply:
I guess auto_install': ['l10n_it', 'account_edi'], means if both 'l10n_it' and 'account_edi' installed, then l10n_it_edi will be installed. However, `account_qr_code_emv` is not only for `l10n_hk`. What should I do in this case?

Reply:
If it's only for `l10n_hk` in stable, you could  auto_install `account_qr_code_emv` if `account` and `l10n_hk` are installed. And then, during the forward-port to master, you could instead make `l10n_hk` and `l10n_my` depend on `account_qr_code_emv` (and remove its auto_install).
Are there any plans for `l10n_my` in stable too ? In that case it's trickier indeed :/

Reply:
no plan for l10n_my in stable at the moment. Right now I only have the specifications for l10n_hk and l10n_sg

Reply:
Then I think the best way would be to:
- in stable, set `account_qr_code_emv`'s auto_install to `l10n_hk` and `account`
- do the forward-port in master
- in master remove the  `account_qr_code_emv`'s auto_install and add `account_qr_code_emv` in `l10n_hk`'s dependencies. Same for `l10n_sg` which will have to be done in master I fear

Reply:
Sure I will do it in forward-port.